tests/drop_test.py

```diff
@@ -0,0 +1,133 @@
+import random
+from typing import NoReturn
+
+from lynx.common.actions.action import Action
+from lynx.common.actions.drop import Drop
+from lynx.common.object import Object
+from lynx.common.player import Player
+from lynx.common.scene import Scene
+from lynx.common.vector import Vector
+from lynx.common.actions.update_resources import UpdateResources
+
+class TestDropSerialization:
+    expected_serialization_drop = '{"type": "Drop", "attributes": {"object_id": 1, "target_position": {"x": 1, "y": 0}}}'
+
+    def test_success_serialization(self) -> NoReturn:
+        serialized_drop = Drop(object_id=1, target_position=Vector(1, 0)).serialize()
+
+        assert self.expected_serialization_drop == serialized_drop
+
+    def test_success_deserialization(self):
+        expected_drop = Drop(object_id=1, target_position=Vector(1, 0))
+        dummy_drop = Drop.deserialize(self.expected_serialization_drop)
+
+        assert dummy_drop == expected_drop
+
+class TestDropApply:
+    def test_success_apply(self):
+        random.seed(1222)
+        expected_scene = Scene(players=[Player(player_id="dummy", player_resources={"Wood": 0, "Stone": 0}, drop_area=Vector(5, 5))])
+        expected_dummy_object = Object(id=1, name="dummy", owner="dummy", position=Vector(5, 5))
+        expected_dummy_drop = Drop(target_position=Vector(5, 6), object_id=1)
+        expected_dummy_object2 = Object(id=expected_scene.generate_id(), name="Wood", position=Vector(5, 6),
+                                        tags=['pushable', 'pickable'])
+        expected_scene.add_entity(expected_dummy_object)
+        expected_scene.add_entity(expected_dummy_drop)
+        expected_scene.add_entity(expected_dummy_object2)
+
+        random.seed(1222)
```

Comment:
Do we really need to call it twice in one function? Call it in-class e.g.
```
class TestDropApply:
   random.seed(1222)
```

Reply:
Yes, because we ned to reset seed. Without this test will fail.